pages/accused-witches-occupations.vue

```diff
@@ -0,0 +1,59 @@
+    </div>
+    <h1 class="text-sm md:text-xl lg:text-2xl">Occupations without Unknown Values</h1>
+    <iframe style="width: 80vw; height: 70vh; border: none;" src="https://query.wikidata.org/embed.html#%23People%20Associated%20%0A%23defaultView%3ABubbleChart%0ASELECT%20%3FoccupationLabel(COUNT(%3Fperson)%20as%20%3Fcount)%20%0AWHERE%20%0A%7B%0A%20%20%3Fitem%20wdt%3AP4478%20%3Fperson.%0A%20%0A%20%20%3Fitem%20wdt%3AP106%20%3Foccupation.%20%0A%20%20%0A%20%20%0A%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22%5BAUTO_LANGUAGE%5D%2Cen%22.%20%7D%0A%7D%0AGROUP%20BY%20%3FoccupationLabel%0A%0AORDER%20BY%20DESC%20(%3Fcount)" referrerpolicy="origin" sandbox="allow-scripts allow-same-origin allow-popups"></iframe>
+  </div>
+</template>
+
+<script>
+
+import Swal from 'sweetalert2';
+
+definePageMeta({
+   layout: 'default'
+ })
+
+ export default {
+   data: () => ({
+
+   }),
+   methods: {
+     showPageInfo(){
+       Swal.fire({
+         title: 'Occupation Bubble Charts',
+         html: '<div>These bubble charts show the different occupations recorded for the accused witches from within the database using the Wikidata Query Service. There is no occupation recorded for most of the witches listed in the Survey, for two reasons. The first reason is that for many of the witches we have only limited information of any kind about them. The second and more interesting reason concerns the way in which occupations were understood. Most people in early modern Scotland were not known as having a distinct occupation, because most people, most of the time, were engaged in farming. It would have been meaningless to call someone a farmer. But it was also meaningless to designate someone as having an occupation other than farmer unless that was recognisably their main occupation. And some people had several occupations. <b class="font-bold">The first bubble chart includes people with no recorded occupation while the second chart only includes the recorded occupations</b></div>',
```

Comment:
```suggestion
         html: '<div>These bubble charts show the different occupations recorded for the accused witches from within the database using the Wikidata Query Service. There is no occupation recorded for most of the witches listed in the Survey, for two reasons. The first reason is that for many of the witches we have only limited information of any kind about them. The second and more interesting reason concerns the way in which occupations were understood. Most people in early modern Scotland were not known as having a distinct occupation, because most people, most of the time, were engaged in farming. It would have been meaningless to call someone a farmer. But it was also meaningless to designate someone as having an occupation other than farmer unless that was recognisably their main occupation. And some people had several occupations. <strong>The first bubble chart includes people with no recorded occupation while the second chart only includes the recorded occupations</strong></div>',
```